subfun/BIDS_RFX.m

```diff
@@ -271,7 +271,7 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
             matlabbatch{j}.spm.stats.factorial_design.masking.tm.tm_none = 1;
             matlabbatch{j}.spm.stats.factorial_design.masking.im = 1;
             matlabbatch{j}.spm.stats.factorial_design.masking.em = {...
-                fullfile(RFX_FolderName,'Meanmask.nii')};
+                fullfile(RFX_FolderName,'MeanMask.nii')};
             matlabbatch{j}.spm.stats.factorial_design.globalc.g_omit = 1;
```

Comment:
Linux is more stubborn about capital letter. So I corrected the name of the mask